implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -329,12 +335,16 @@ async fn run_background_node_impl(
         Some(get_identity_override(ctx, &cfg).await?)
     };
 
+    if let Some(path) = &c.project {
```

Comment:
Have you tested it? I may be wrong, but that looks like it shouldn't work, because `project` value is not forwarded in `startup::spawn_node`

Reply:
Changed in https://github.com/build-trust/ockam/pull/3336/commits/8a1b8744d23f46fc8d3bd454471a902b3933a099